ophyd/signal.py

```diff
@@ -612,6 +644,33 @@ def wait_for_connection(self, timeout=1.0):
 
     @property
     @raise_if_disconnected
+    def tolerance(self):
+        '''The tolerance of the write PV, as reported by EPICS
+
+        Can be overidden by the user at the EpicsSignal level.
+
+        Returns
+        -------
+        tolerance : float or None
+        Using the write PV's precision:
+            If precision == 0, tolerance will be None
+            If precision > 0, calculated to be 10**(-precision)
+        '''
+        if self._tolerance is not None:
```

Comment:
`tolerance.setter` merely sets `self._tolerance`, but it is probably better practice/less confusing to set the internal value